client/verta/verta/environment/_python.py

```diff
@@ -29,30 +31,44 @@ class Python(_environment._Environment):
         captured.
     env_vars : list of str, optional
         Names of environment variables to capture. If not provided, nothing will be captured.
+    apt_packages : list of str, optional
+        Apt packages to be installed alongside a Python environment
     _autocapture : bool, default True
         Whether to enable the automatic capturing behavior of parameters above.
 
     Attributes
     ----------
     requirements : list of str
         pip requirements.
+    apt_packages : list of str
+        apt packages to be installed alongside a Python environment
```

Comment:
In addition to the period, "apt"'s capitalization here should match the above "Apt".\
(I wrote "pip" in lowercase because that's how the project stylizes its own name, but maybe that's just completely bananas).
```suggestion
        apt packages to be installed alongside a Python environment.
```